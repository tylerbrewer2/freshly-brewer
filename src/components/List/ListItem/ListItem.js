import React from 'react';
import styled from 'styled-components';

const ListItem = styled.li`
  font-family: 'AvenirNextRegular', 'Helvetica', 'sans-serif';
  font-size: 12px;
  line-height: 30px;
`

export default ListItem;
