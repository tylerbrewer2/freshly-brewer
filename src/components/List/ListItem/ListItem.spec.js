import React from 'react';
import renderer from 'react-test-renderer';

import ListItem from './ListItem';

it('renders correctly', () => {
  const tree = renderer.create(
    <ListItem>Test item</ListItem>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
