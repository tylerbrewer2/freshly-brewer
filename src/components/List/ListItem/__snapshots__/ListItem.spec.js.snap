exports[`test renders correctly 1`] = `
<li
  className="sc-bdVaJa dfPjrf">
  Test item
</li>
`;
